test/integration/attachment.test.js

```diff
@@ -76,4 +84,17 @@ describe('attachments', function () {
     expect(response.status).to.equal(400)
     expect(response.error.text).to.equal('Unexpected token t in JSON at position 0')
   })
+
+  it('should return from the JSON route', async function () {
```

Comment:
Also need error case for if an `application/json` response is requested but the file is not `json`